single-node/run-performance-tests.sh

```
@@ -49,6 +49,11 @@ function after_execute_test_scenario() {
     download_file "$wso2is_host_alias" $is_home/repository/logs/wso2carbon.log "$wso2is_host_alias.log"
     download_file "$wso2is_host_alias" $is_home/repository/logs/gc.log $wso2is_host_alias"_gc.log"
     download_file "$wso2is_host_alias" $is_home/repository/logs/heap-dump.hprof "$wso2is_host_alias-heap-dump.hprof"
+
+    local jfr_file="$is_home/repository/logs/recording.jfr"
+    ssh "$wso2is_host_alias" "IS_PID=\$(pgrep -u ubuntu java | head -1) && JCMD=\$(dirname \$(readlink -f /proc/\${IS_PID}/exe))/jcmd && echo \"Dumping JFR for PID: \${IS_PID}\" && \${JCMD} \${IS_PID} JFR.dump name=perf-recording filename=${jfr_file}" \
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)